airflow/www/views.py

```diff
@@ -1267,98 +1267,6 @@ def code(self, dag_id, session: Session = NEW_SESSION):
             wrapped=conf.getboolean("webserver", "default_wrap"),
         )
 
-    @expose("/dag_details")
```

Comment:
We're missing the redirect for this URL and the legacy one.

Reply:
Will revert this

Reply:
I have reverted the changes! Not much idea on the legacy side. Would need your help with the removal of relevant parts